crengine/src/lvrend.cpp

```diff
@@ -3518,7 +3536,7 @@ void renderFinalBlock( ldomNode * enode, LFormattedText * txform, RenderRectAcce
                     << baseflags << ")\n";
             #endif
 
-            ldomNode * parent = enode->getParentNode();
+            ldomNode * const parent = enode->getParentNode();
```

Comment:
Just curious (I'm an amateur at C++):
- is `ldomNode * const parent` exactly the same as `const ldomNode * parent` ? If yes, please put the `const` at start, like I'm used to see it elsewhere (and you did it above in `const int em`) so there's no doubt :)
- how/why do you decide to add `const` ? Is this a reflex ? Do you have to look aroud to be sure it's not updated ? Or do you add it by default and see if the compiler complains?

Reply:
>     * is `ldomNode * const parent` exactly the same as `const ldomNode * parent` ? If yes, please put the `const` at start, like I'm used to see it elsewhere (and you did it above in `const int em`) so there's no doubt :)

It isn't, c.f., [the clockwise/spiral rule](http://c-faq.com/decl/spiral.anderson.html) ;).

(Here, the pointer itself is constant, not the data it points to).